src/NuGet.Clients/PackageManagement.UI/UserInterfaceService/NuGetUI.cs

```diff
@@ -163,17 +163,9 @@ public bool ForceRemove
 
         public PackageIdentity SelectedPackage { get; set; }
 
-        public void RefreshPackageStatus()
+        public void OnActionsExecuted(IEnumerable<ResolvedAction> actions)
         {
-            if (PackageManagerControl != null)
-            {
-                UIDispatcher.Invoke(() => { PackageManagerControl.UpdatePackageStatus(); });
-            }
-
-            if (_detailControl != null)
-            {
-                UIDispatcher.Invoke(() => { _detailControl.Refresh(); });
-            }
+            this._context.SolutionManager.OnActionsExecuted(actions);
```

Comment:
PackageManagerControl listens to ActionsExecutedEvent. Firing the event will cause PackageManagerControl to refresh.